config/src/lib.rs

```diff
@@ -1720,7 +1720,7 @@ impl Default for Config {
             allow_paths: vec![],
             include_paths: vec![],
             force: false,
-            evm_version: Default::default(),
+            evm_version: EvmVersion::London,
```

Comment:
Shouldn't this also be Paris or Shanghai?

Reply:
yep yep indeed. I think Shanghai would be best as it's gonna be the new default for other tooling (and `EvmVersion` defaults to Paris)

Reply:
@Evalir you prob want to upstream evm version default change to ethers

Reply:
nvm, saw the comments below